static/app/utils/getKeyCode.ts

```diff
@@ -0,0 +1,79 @@
+/* key maps and utils retrieved from  https://github.com/jaywcjlove/hotkeys */
```

Comment:
nit, I would make this a regular `//` comment. We rarely use `/*` style comments in the codebase (would be nice if we linted on this
```suggestion
// key maps and utils retrieved from https://github.com/jaywcjlove/hotkeys
```